js/ui/appIconBar.js

```diff
@@ -1004,6 +1036,24 @@ const AppIconBar = new Lang.Class({
         Main.overview.connect('showing', Lang.bind(this, this._updateActiveApp));
         Main.overview.connect('hidden', Lang.bind(this, this._updateActiveApp));
 
+
+        let keybinding_settings = new Gio.Settings({ schema: SHELL_KEYBINDINGS_SCHEMA });
```

Comment:
Heh I can context switch between c and js styles pretty good these days. But I guess different js is beyond me